locations/spiders/piggly_wiggly_us.py

```diff
@@ -1,11 +1,25 @@
-from locations.storefinders.freshop import FreshopSpider
+import json
+import re
 
+from scrapy.spiders import SitemapSpider
 
-class PigglyWigglyUSSpider(FreshopSpider):
+from locations.dict_parser import DictParser
+
+
+class PigglyWigglyUSSpider(SitemapSpider):
     name = "piggly_wiggly_us"
     item_attributes = {"brand": "Piggly Wiggly", "brand_wikidata": "Q3388303"}
-    app_key = "piggly_wiggly_nc"
+    sitemap_urls = ["https://www.pigglywiggly.com/wp-sitemap-posts-page-1.xml"]
+    sitemap_rules = [(r"/store-locations/.+/", "parse")]
+
+    def parse(self, response, **kwargs):
+        locations = json.loads(re.search(r"locations = (\[?{.+}\]?);", response.text).group(1))
+        if isinstance(locations, dict):
+            locations = locations.values()
 
-    def parse_item(self, item, location):
-        item["phone"] = item["phone"].split("|", 1)[0].split("\n", 1)[0].strip()
-        yield item
+        for location in locations:
+            item = DictParser.parse(location)
+            item["ref"] = location["storeNumber"]
+            item["website"] = item.pop("email")
```

Comment:
It looks like they have store-specific URLs available e.g. https://www.shopthepig.com/stores/sister-bay-95/5431. Can you try to include those?

Reply:
I don't think they were available, a couple had them in the name. I can check again when I get home, or tomorrow.

Reply:
Looks like there are multiple companies operating stores under the "Piggly Wiggly" brand. https://www.thepig.net/store-locations/ does some, https://www.shopthepig.com/ does Wisconsin, etc. They all seem to have their own store locators, so it's probably ok to keep what we have for now.